trunk/3rdparty/st-srs/stk.c

```diff
@@ -153,7 +175,6 @@ static char *_st_new_stk_segment(int size)
 
 
 /* Not used */
-#if 0
 void _st_delete_stk_segment(char *vaddr, int size)
```

Comment:
1. remove the no longer correct `/* Not used */` comment.
2. `_st_delete_stk_segment` only used when `#ifndef MD_CACHE_STACK`, but when `#ifdef MD_CACHE_STACK`, is the stk segment released? or where to call `_st_delete_stk_segment`?

Reply:
Fixed.

If stack is cached, won't call this function.